src/core_modules/capture-core/components/Pages/shared-contexts.ts

```diff
@@ -0,0 +1,6 @@
+import { createContext } from 'react';
+
+export type ResultsPageSizeContextType = { resultsPageSize: number };
```

Comment:
Use the same variable name as it used to be. `InitialValueOfResultsPageSizeContext`